pkg/restclient/device.go

```diff
@@ -43,3 +43,11 @@ func (c *Client) DeviceEdit(peerid, device, state string) error {
 	err := c.post(url, req, http.StatusOK, nil)
 	return err
 }
+
+// DevicesInPeer lists devices in peer
+func (c *Client) DevicesInPeer(peerid string) (deviceapi.ListDeviceResp, error) {
```

Comment:
why this is needed? 
we already have an API to do this functionality. https://github.com/gluster/glusterd2/blob/master/pkg/restclient/device.go#L26